release/release_tests.yaml

```diff
@@ -3643,7 +3643,7 @@
       num_nodes: 15
 
 
-- name: pipelined_data_ingest_benchmark
+- name: pipelined_data_ingest_benchmark_200gb
```

Comment:
would this break dashboard @scottjlee?

Reply:
Yes I believe so, since this field is used as the filter on `name` in the query used to generate the `pipelined_data_ingest_benchmark` chart. If we are OK with either removing benchmark data for this particular test prior to changing this name, then we can go ahead with this, and I can update the query. Otherwise, it _may_ be possible to use two queries pulling from both names and `UNION` them, but I don't have enough experience with Preset yet to say for sure.

Reply:
I think you can just use a `OR` condition:

```
AND (name = "pipelined_data_ingest_benchmark" OR name = "pipelined_data_ingest_benchmark_200gb")
```

Reply:
@scottjlee - then let's update dashboard after this PR is merged, thanks.

Reply:
Thanks for taking care of dashboard part. Btw, this is now "_1tb", to make it more comparable with the streaming ingest benchmark.